### PR TITLE
LPD-32389 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://liferay.atlassian.net/browse/LPD-32389

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ProjectController.java [hunk 1]
```diff
@@ -537,6 +540,28 @@ public List<ProjectDisplay> getProjects() {
 			});
 	}
 
+	@GET
+	@Path("/usage")
+	@RolesAllowed(RoleConstants.SITE_ADMINISTRATOR)
+	public List<ProjectUsageDisplay> getProjects(
```
Comment (rachaelkoestartyo) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ProjectController.java:546]: rename method to `getProjectUsages`

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ProjectController.java [hunk 2]
```diff
@@ -537,6 +540,28 @@ public List<ProjectDisplay> getProjects() {
...
+		@FormParam("groupId") Long groupId) {
```
Comment (rachaelkoestartyo) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ProjectController.java:547]: let's use `@QueryParam`

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ProjectController.java [hunk 3]
```diff
@@ -537,6 +540,28 @@ public List<ProjectDisplay> getProjects() {
...
+
+		List<FaroProject> faroProjects = new ArrayList<>();
+
+		if (Validator.isNotNull(groupId)) {
+			faroProjects.add(
+				_faroProjectLocalService.fetchFaroProjectByGroupId(groupId));
+		}
+		else {
+			faroProjects.addAll(
```
Comment (rachaelkoestartyo) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ProjectController.java:556]: no need to addAll, set `faroProjects = _faroProjectLocalService.getFaroProjects(QueryUtil.ALL_POS, QueryUtil.ALL_POS));`

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Review by rachaelkoestartyo — APPROVED** [review]

**Comment by marcosapmf** [comment]
ci:forward

**Pull request closed without merging**